### ISSUE TITLE
Add a "trending topics" display to Activity feed

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#1677]
We should add a new section to the activity template (see the [Activity section of this page](http://publiclab.org/research)); it'd look like this:

![screenshot 2017-10-11 at 6 00 41 pm](https://user-images.githubusercontent.com/24359/31469445-86708cf8-aeaf-11e7-80b6-527206fee631.png)

We could do this using Bootstrap label styling, like:

```html
<p>
  <span style="font-size: 18px;">
  <span class="label label-primary">tagname</span>
  <span class="label label-primary">tagname</span>
  <span class="label label-primary">tagname</span>
  </span>
</p>
```

That could live right above this line:

https://github.com/publiclab/plots2/blob/master/app/views/dashboard/_activity.html.erb#L51

We'd need a new method to drive this, under `Tag.trending` -- a new method in the Tag model, here:

https://github.com/publiclab/plots2/blob/master/app/models/tag.rb

```ruby
def trending(limit = 5, time_ago = 1.month)
   Tag.joins(:node_tag, :node)
               .select('node.nid, node.status, term_data.*, community_tags.*')
               .where('node.status = ?', 1)
               .where('community_tags.date > ?', (DateTime.now - time_ago).to_i)
               .group(:name)
               .order('count DESC')
               .limit(limit)
```

Since this is a big query to run, we should also cache this so that it's recalculated every day, using a similar strategy to:

```html
  <% cache('trending-tags', expires_in: 24.hours) do %>
    <!-- trending tags code in here -->
  <% end %>
```

**Comment by sagarpreet-chadha** [comment]
Sir...can i try this ?

**Comment by jywarren** [comment]
Yes please! That would be super. It's a bit more complex so please ask if
you get in trouble!

**Comment by jywarren** [comment]
Solved in #1684 ! Thanks!!
